config.yml

```diff
@@ -17,6 +17,10 @@ uvc_gadget_repo_version: master
 # Whether to enable piwebcam at boot or not.
 piwebcam_boot_enabled: true
 
+# PI username. Default: User running the playbook
+username: "{{ lookup('env', 'USER') }}"
```

Comment:
Can you leave the default `pi`? And we can add a line in the README somewhere mentioning to change it if you aren't using a `pi` user.

Also, just one empty line after this variable (this PR adds two newlines).